### ISSUE TITLE
Android Studio 3.2 Canary 2 plugin failed

### ISSUE THREAD

**Issue by mtrakal** [pbreault/adb-idea#85]
Android Studio 3.2 Canary 2
Build #AI-173.4591728, built on February 7, 2018
JRE: 1.8.0_152-release-1024-b02 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0

```
org.jetbrains.android.facet.AndroidFacet.getAndroidModel()Lcom/android/tools/idea/model/AndroidModel;
java.lang.NoSuchMethodError: org.jetbrains.android.facet.AndroidFacet.getAndroidModel()Lcom/android/tools/idea/model/AndroidModel;
	at com.developerphil.adbidea.adb.DeviceResultFetcher.fetch(DeviceResultFetcher.kt:20)
	at com.developerphil.adbidea.adb.AdbFacade.executeOnDevice(AdbFacade.java:73)
	at com.developerphil.adbidea.adb.AdbFacade.clearDataAndRestart(AdbFacade.java:61)
	at com.developerphil.adbidea.action.ClearDataAndRestartAction.actionPerformed(ClearDataAndRestartAction.java:10)
	at com.developerphil.adbidea.action.AdbAction.actionPerformed(AdbAction.java:13)
	at com.intellij.openapi.actionSystem.ex.ActionUtil$1.run(ActionUtil.java:220)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:237)
	at com.intellij.ui.popup.PopupFactoryImpl$ActionPopupStep.performAction(PopupFactoryImpl.java:966)
	at com.intellij.ui.popup.PopupFactoryImpl$ActionPopupStep.performAction(PopupFactoryImpl.java:956)
	at com.intellij.ui.popup.PopupFactoryImpl$ActionPopupStep.lambda$onChosen$0(PopupFactoryImpl.java:950)
	at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:195)
	at com.intellij.ui.popup.AbstractPopup.lambda$null$7(AbstractPopup.java:1431)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:762)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:732)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:822)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:650)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:366)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by pbreault** [comment]
This should have been fixed in version 1.5.1. 
Can you update the plugin and validate if it fixes the issue?

**Comment by mtrakal** [comment]
It's working again. Thnaks a lot 🥇
